src/Core/Animation/Handle/Skeleton.cpp

```diff
@@ -239,18 +239,18 @@ std::ostream& operator<<( std::ostream& os, const Skeleton& skeleton ) {
                       : skeleton.m_graph.isBranch( i )
                             ? "BRANCH"
                             : skeleton.m_graph.isLeaf( i ) ? "LEAF" : "UNKNOWN";
-        const int pid = skeleton.m_graph.m_parent.at( i );
+        const int pid = skeleton.m_graph.parents().at( i );

```

Comment:
what is the objective of using `.at(i)` here and not `[i]`. Do we really need to do bound-checking at each access ?
The same remarks apply to several places below in `src/Core/Animation/Handle/Skeleton.cpp`

Reply:
I do agree.
But I have not processed this file yet (only BVH and AdjacencyList + the side effects).

Reply:
OK. This was just to ensure that you'll be attentive to this kind of stuff ;)